_posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md

```diff
@@ -23,7 +23,7 @@ Think about the following: How many developers of the dapps you are using do you
 
 Let’s imagine you want to interact with a dapp using MetaMask and it shows you this dialogue: 
 
-![Yolo Signing](../img/2020/05/MetaMask_Yolosign.png)
+![Yolo Signing](https://raw.githubusercontent.com/ethereum/solidity-blog/67a114759bcec663a094814e813e32d2e7d164f1/img/2020/05/MetaMask_Yolosign.png)
```

Comment:
Still the easiest (not depending on github) is: `/img/2020/...` as mentioned 2 PRs ago :)

Reply:
but it didn't work? wasn't that what I tried just before?

Reply:
You guys tried `./` and `../`. I'm saying `/`.

Reply:
I didn't try `./` but yes doh, I missed the `/` here https://github.com/ethereum/solidity-blog/pull/60. Didn't see your comment, sorry! I prefer only the `/`. Let's change it into that.